Scripts/Setup/setupExperiment.sh

```diff
@@ -160,6 +160,7 @@ DELT='DEFAULT' # time decrement for auto restart (DEFAULT: select according to t
 DATADIR='' # root directory for data
 DATATYPE='CESM' # boundary forcing type
 CMIP6MODEL='MPI-ESM1-2-HR' # CMIP6 model.
+CMIP6MDLVER='PROJ' # CMIP6 model version.
```

Comment:
I would use lower case, but I think we concluded that we don't want to introduce a new variable here and just treat it as part of the CMIP6MODEL string, since not all models may even need that...?

Reply:
Fixed.